examples/0-isbsvc-redis-cluster.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: InterStepBufferService
+metadata:
+  name: default
+spec:
+  redis:
+    external:
+      # deploy the redis cluster on k8s:
+      #     helm repo add bitnami https://charts.bitnami.com/bitnami
+      #     helm install numaflow-redis-cluster --set image.tag=6.2 --set usePassword=false bitnami/redis-cluster --version 8.1.1
+      # note:
+      #     Need to provide at least 2 addresses because we use the universal client in the shared redis client code.
+      url: "numaflow-redis-cluster-0.numaflow-redis-cluster-headless.default.svc.cluster.local:6379,numaflow-redis-cluster-1.numaflow-redis-cluster-headless.default.svc.cluster.local:6379"
```

Comment:
Maybe use `numaflow-redis-cluster-0.numaflow-redis-cluster-headless:6379,numaflow-redis-cluster-1.numaflow-redis-cluster-headless:6379,numaflow-redis-cluster-2.numaflow-redis-cluster-headless:6379` - without having the namespace in it.